pkgs/by-name/po/ponyc/package.nix

```diff
@@ -67,9 +70,7 @@ stdenv.mkDerivation (rec {
       ./disable-networking-tests.patch
     ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
-      (replaceVars ./fix-darwin-build.patch {
-        libSystem = darwin.Libsystem;
```

Comment:
There is a merge conflict on this line

Reply:
Indeed, just saw the merge come across. Fixing it up.